utils/rpms/daos.spec

```diff
@@ -157,19 +157,10 @@ scons %{?no_smp_mflags}              \
       PREFIX=%{?buildroot}%{_prefix}
 BUILDROOT="%{?buildroot}"
 PREFIX="%{?_prefix}"
-sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/TESTING/.build_vars.*
-mv %{?buildroot}%{_prefix}/lib{,64}
-#mv %{?buildroot}/{usr/,}etc
-mkdir -p %{?buildroot}/%{_exec_prefix}/lib/%{name}
-mv %{?buildroot}%{_prefix}/lib64/daos %{?buildroot}/%{_exec_prefix}/lib/
-mv %{?buildroot}%{_prefix}/{TESTING,lib/%{name}/}
-cp -al ftest.sh src/tests/ftest %{?buildroot}%{daoshome}/TESTING
-find %{?buildroot}%{daoshome}/TESTING/ftest -name \*.py[co] -print0 | xargs -r0 rm -f
-#ln %{?buildroot}%{daoshome}/{TESTING/.build_vars,.build_vars-Linux}.sh
-mkdir -p %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/daos_api.py %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/daos_cref.py %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/conversion.py %{?buildroot}%{daoshome}/utils/py
+sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{daoshome}/TESTING/.build_vars.*
+sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/lib/.build_vars.*
```

Comment:
Why do we have two sets of `.build_vars.*` files?

Reply:
The location $PREFIX/TESTING is generated by cart, not sure if it's safe to make any changes there:
./_build.external/cart/SConstruct:    env.InstallAs('$PREFIX/TESTING/.build_vars.sh', sh_build_vars)
./_build.external/cart/SConstruct:    env.InstallAs('$PREFIX/TESTING/.build_vars.json', json_build_vars)

Reply:
When DAOS is being built as an RPM, cart is not built so there shouldn't be anything under `install/` from cart.  You are probably confusing what gets put into `install/` during an RPM build vs. a full daos-stack build.